### ISSUE TITLE
No deleted posts fetched from 3rd party archives ever loads out and shows up

### ISSUE THREAD

**Issue by Impeta** [TuxedoTako/4chan-xt#5]
```
Uncaught TypeError: templates[i] is undefined
    cat moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:322
    comment moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:7168
    parseArchivedPost moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:7173
    archivedPost moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:7124
    onloadend moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:21235
    execTask moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:21474
138399076:322:15
    cat moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:322
    comment moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:7168
    parseArchivedPost moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:7173
    archivedPost moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:7124
    onloadend moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:21235
    execTask moz-extension://2b246608-699a-44b7-936d-918c20a8e8ad/ 4chan XT.user.js#13:21474
```

This is what shows up on Firefox's console, whenever you hover your pointer over replying post numbers quoting to deleted posts off a thread. It always stays on "Loading post No.XXXXXXXX" and never displays back fetched archived posts.

**Comment by TuxedoTako** [comment]
Can you provide a link to such a thread? I don't run into threads with deleted posts often on the boards I browse.

**Comment by TuxedoTako** [comment]
Should be fixed in 2.1.2.
